room/compiler-processing/src/main/java/androidx/room/compiler/processing/XProcessingEnv.kt

```diff
@@ -33,6 +33,12 @@ import kotlin.reflect.KClass
 interface XProcessingEnv {
 
     val backend: Backend
+
+    /**
+     * The target language the processor is set to expect
```

Comment:
nit: Reword a bit to clarify: `The expected language of the sources generated by the processor.`

Reply:
Changed the description